src/enums/filing-types.ts

```diff
@@ -3,7 +3,7 @@ export enum FilingTypes {
   ALTERATION = 'alteration',
   AGM_EXTENSION = 'agmExtension',
   AGM_LOCATION_CHANGE = 'agmLocationChange',
-  AMALGAMATION = 'amalgamation',
+  AMALGAMATION = 'amalgamationApplication',
```

Comment:
Do you think the key should be renamed as well?

Reply:
(The UIs will need to import the updated shared enum anyway.)

Reply:
Was thinking about that. Yes, I think that would be a good idea. On it!